internal/setup/job.go

```diff
@@ -101,6 +101,14 @@ func NewJob(t *redskyv1beta1.Trial, mode string) (*batchv1.Job, error) {
 			},
 		}
 
+		if len(c.Args) == 0 {
+			c.Args = []string{mode}
+		}
+
+		if len(task.Command) > 0 {
```

Comment:
I feel like this should be `len(task.Command) > 0 && c.Image != ""` to help prevent changing the entrypoint on the default setup tools image (this check would not preclude explicitly specifying the setup tools image by name).